### DISCUSSION TITLE
Toggle margin images layout css width on click.

### DISCUSSION THREAD

**Discussion by OrenBochman** [quarto-dev/quarto-cli#5530] [Q&A] [answered]
Hi, I am making a book using Quarto with course notes on Bayesian stats. I have many screenshots in the margin. 

via say

![slide](images/ss-c2l04-01-Metropolis-Hastings.png){.column-margin .mag}

I'd like the HTML user to be able to click the screenshot and get it to expand to full screen or at least to 
.column-page-inset or something like it. Toggling the margin is even better.

I tried to add a .mag with some custom CSS for the, but this was either bad CSS or more likely gets ignored. 
A small CSS and or js example for this in the docs would go a long way.

Thanks!

**Comment by mcanouil** [comment, marked as answer]
See <https://github.com/quarto-ext/lightbox>.
